testing/test_ASH.py

```diff
@@ -162,6 +163,10 @@ async def test_botAssignment(self):
             AntiSpamHandler(False)
 
         AntiSpamHandler(commands.Bot(command_prefix="!"))
+        AntiSpamHandler(commands.AutoShardedBot(command_prefix="!"))
+        discord.Client()
```

Comment:
You're not testing anything with this line?
Did you mean `AntiSpamHandler(discord.Client())`